.builds/freebsd.yml

```diff
@@ -19,26 +19,36 @@ sources:
 environment:
   SOURCEHUT: 1
   LANG: en_US.UTF-8
+  CMAKE_EXTRA_FLAGS: -DTRAVIS_CI_BUILD=ON -DMIN_LOG_LEVEL=3
 
 tasks:
 - build-deps: |
     cd neovim
     gmake deps
-- build-debug: |
-    mkdir -p neovim/build
-    cd neovim/build
-    cmake -G Ninja -DTRAVIS_CI_BUILD=ON -DCMAKE_BUILD_TYPE=Debug -DMIN_LOG_LEVEL=3 ..
-    ninja -v nvim unittest-prereqs functionaltest-prereqs
-- test-debug: |
-    cd neovim/build
-    ninja -v unittest functionaltest
-- clean-build: |
-    rm -rf build
 - build-release: |
-    mkdir -p neovim/build
-    cd neovim/build
-    cmake -G Ninja -DTRAVIS_CI_BUILD=ON -DCMAKE_BUILD_TYPE=Release -DMIN_LOG_LEVEL=3 ..
-    ninja -v nvim unittest-prereqs functionaltest-prereqs
+    cd neovim
+    gmake CMAKE_BUILD_TYPE=Release CMAKE_EXTRA_FLAGS="${CMAKE_EXTRA_FLAGS}" nvim
 - test-release: |
-    cd neovim/build
-    ninja -v unittest functionaltest
+    cd neovim
+    gmake CMAKE_BUILD_TYPE=Release CMAKE_EXTRA_FLAGS="${CMAKE_EXTRA_FLAGS}" unittest functionaltest
```

Comment:
No need to pass cmake stuff here.

Reply:
Yeah... copy and past for the moment.  I'll clean it up before calling it done.